### PR TITLE
Fix CPU coercion in SFS backend

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: supportedBackends/sfs/src/test/scala/cromwell/backend/sfs/config/DeclarationValidationSpec.scala [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+class DeclarationValidationSpec extends FlatSpec with Matchers with TableDrivenPropertyChecks {
+  behavior of "DeclarationValidation"
+  
+  def validateCpu(key: String) = {
+    val expression = WdlExpression.fromString("5")
+    val declarationValidation = DeclarationValidation.fromDeclaration(Declaration(WomIntegerType, key, Option(expression), None, null))
+    declarationValidation.extractWdlValueOption(ValidatedRuntimeAttributes(Map(key -> refineMV[Positive](5)))) shouldBe Some(WomInteger(5))
+  }
+  
+  it should "validate cpu attributes" in {
```
Comment (cjllanwarne) [inline supportedBackends/sfs/src/test/scala/cromwell/backend/sfs/config/DeclarationValidationSpec.scala]: ToL: If you moved the `it should` inside the `forAll` you could have individual test cases and individual pass/fail on each attribute

File: supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/CpuDeclarationValidation.scala [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package cromwell.backend.impl.sfs.config
+
+import cromwell.backend.validation.{RuntimeAttributesValidation, ValidatedRuntimeAttributes}
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric.Positive
+import wdl.draft2.model.Declaration
```
Comment (cjllanwarne) [inline supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/CpuDeclarationValidation.scala]: ToL: It obviously compiles so I guess so, but is this really a `WDL` `Declaration`?

File: supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/CpuDeclarationValidation.scala [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import wom.values.WomValue
+
+class CpuDeclarationValidation(declaration: Declaration, instanceValidation: RuntimeAttributesValidation[Int Refined Positive]) extends DeclarationValidation(declaration, instanceValidation) {
+  override def extractWdlValueOption(validatedRuntimeAttributes: ValidatedRuntimeAttributes): Option[WomValue] = {
```
Comment (cjllanwarne) [inline supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/CpuDeclarationValidation.scala]: ToL: not the right time but... `extractWomValueOption`?

File: supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/CpuDeclarationValidation.scala (hunk 2 above)
Comment (Horneth) [reply supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/CpuDeclarationValidation.scala]: Yep the config backend uses draft 2 wdl

**Comment by Horneth** [comment]
Since this is blocking the release and comments are ToL I'm going to merge as is and address the comments in a later PR.
@mcovarr Yeah we could also do that, I just never know how much coercion logic from third-party libraries to jam into WOM (e.g now we coerce from spray.json but not circe.json)

**Pull request merged**